### ISSUE TITLE
Duplicate morphs in menu

### ISSUE THREAD

**Issue by AshleyZorah** [Budschie/BudschieMorphMod#29]
I'm trying to use the morph mod but it seems to have accumulated multiple duplicates of standard mobs such as zombies and phantoms, making it almost impossible to use as there are almost hundreds at this point.
![2021-09-27_22 56 15](https://user-images.githubusercontent.com/91503904/134990905-df8af9a9-085d-4f8e-8aca-00ec6c3d051f.png)
Just as proof that it's happening, it looks like it happens when they're at different stages of animation, perhaps something to do with the NBTs?

**Comment by Budschie** [comment]
This is strange... These duplicate mobs are not in a row, which is weird, as entities of the same entity type get grouped together in the same row normally. I suppose that you are using the newest version of the mod (2.0.1), right? If so, this means that the displayed entities at the right side are in fact not the same entity, but have different entity types. (To explain a bit of terminology, the id of an entity type is the thing that you use to tell the "/summon" command what entity it should summon. So, "minecraft:zombie" would be an example of such entity type id. A baby zombie doesn't get its own entity id, but rather uses the entity id of the adult zombie, so a baby zombie would be created with the same entity type, but with different NBT data that tells the game that this specific zombie is a baby zombie).

So, to put it simply, if you use the newest version of the mod, it is likely that the morphs shown on the right are two distinct entities (for some reason).

But it could also be an edge case bug too, so, if you want, you can provide your playerdata file (the file containing the saved player data, located in "saves/<your-save>/playerdata/<your-minecraft-UUID>"), it would help me out a lot.

Anyways, thanks for reporting this bug, and for reading this reply. I hope you have great day :)
- Budschie
